### PR TITLE
Update ops notebook

### PR DESCRIPTION
**Pull request by MichaelBroughton**
Added new protocols to list of protocols as well as some minor cleanup.

### CODE REVIEW

File: docs/operators_and_observables.ipynb [hunk 1]
```diff
@@ -99,9 +99,9 @@
...
+    "| Unitary operators | Any class implementing the `_unitary_` and `_has_unitary_` protocol | [Protocols](protocols.ipynb), [Gates and operations](gates.ipynb), [Custom gates](custom_gates.ipynb) | `cirq.Gate` <br> `cirq.X(qubit)` <br> `cirq.CNOT(q0, q1)` <br> `cirq.MatrixGate.on(qubit)` <br> `cirq.Circuit` (if it only contains unitary operations)  |\n",
     "| Measurements | `cirq.measure` and `cirq.MeasurementGate`  | [Gates and operations](gates.ipynb) | `cirq.measure(cirq.LineQubit(0))` |  \n",
-    "| Quantum channels | <ul> <li>Kraus operators (any class implementing the `_kraus_` protocol)</li><li>Pauli basis (cirq.PauliSum)</li> </ul> | [Protocols](protocols.ipynb) | `cirq.DepolarizingChannel(p=0.2)(q0)` <br> `cirq.X.with_probability(0.5)`|"
+    "| Quantum channels | <ul> <li>Kraus operators (any class implementing the `_kraus_` and `_has_kraus_` protocol)</li><li>Unitary mixtures (any class implementing the `_mixture_` and `_has_mixture_` protocol)</li><li>Pauli basis (cirq.PauliSum)</li> </ul> | [Protocols](protocols.ipynb) | `cirq.DepolarizingChannel(p=0.2)(q0)` <br> `cirq.X.with_probability(0.5)`|"
```
Comment (dabacon) [inline docs/operators_and_observables.ipynb:104]: not your doing but how do you create a channel from a Pauli basis?

**Review by dabacon — APPROVED** [review]
One comment about this table (which reviewnb totally barfs on)

File: docs/operators_and_observables.ipynb (hunk 1 above)
Comment (MichaelBroughton) [reply docs/operators_and_observables.ipynb:104]: No idea. It was already there and I think the tutorial talks about PauliSum and expectations briefly which seemed OK to me. Do you think we should remove it ?

Comment (dabacon) [reply docs/operators_and_observables.ipynb:104]: yeah I don't see how it really makes sense here

Comment (MichaelBroughton) [reply docs/operators_and_observables.ipynb]: Done.

**Pull request merged**